synapse/cortex.py

```diff
@@ -280,10 +285,11 @@ def _convert_reqdict(reqdict):
         '''
         crons = []
 
-        for iden, cron in self.cell.agenda.list():
+        isallowed = await self.allowed(('cron', 'get'))
```

Comment:
in trigger listing, we're checking perms for each trigger. This seems very similar but is implemented just once and gates ALL agenda items. feels like a slight dissonance.

Reply:
fixed